packages/webapp/src/components/Crop/CropHeader.jsx

```diff
@@ -5,14 +5,33 @@ import { useTranslation } from 'react-i18next';
 import styles from './styles.module.scss';
 import clsx from 'clsx';
 
-function CropHeader({
-  crop_translation_key,
-  crop_variety_name,
-  supplier,
-  crop_variety_photo_url,
-  onBackClick,
-}) {
+function CropHeader({ variety, onBackClick }) {
   const { t } = useTranslation(['translation', 'crop']);
+  const {
+    crop_variety_name,
+    crop_genus,
+    crop_specie,
+    crop_varietal,
+    crop_cultivar,
+    supplier,
+    crop_variety_photo_url,
+    crop_translation_key,
+  } = variety;
+
+  const crop_scientific_name = () => {
+    const genus = crop_genus
+      ? crop_genus.charAt(0).toUpperCase() + crop_genus.slice(1).toLowerCase() + ' '
+      : '';
+    const species = crop_specie ? crop_specie : '';
+    return `${genus}${species}`;
+  };
+
+  const varietalDetails = () => {
+    const varietal = crop_varietal ? `var. ${crop_varietal.toLowerCase()}` : '';
+    const cultivar = crop_cultivar ? `'${crop_cultivar}'` : '';
+    return `${varietal} ${cultivar}`;
```

Comment:
Nice fix for the scientific name!
It would be great if you could do the same thing here!